### PR TITLE
clarify maintenance status in readme

### PR DESCRIPTION
**Pull request by SethTisue**
@adriaanm is the wording about Lightbend appropriate?

### CODE REVIEW

**Comment by hvesalai** [comment]
I don't think I deserve the title. It's been a while since I made the previous release and actually don't even remember how it would be made currently. The previous release was made by @octonato

**Comment by SethTisue** [comment]
hmm... “a loose assortment of volunteers including @hvesalai”, then? or would you suggest simply “a loose assortment of volunteers”?

(I'm honestly not too familiar with what goes on in this repo)

as for publishing, this work I'm currently doing — making sure this repo is in some sort of reasonably healthy state so that community maintenance can proceed and prospective contributors aren't too put off  — definitely includes making sure we know how to publish. let's cross that bridge when we're ready to release. maybe we just need to document the status quo, or (especially if the status quo turns out to be broken enough that work is needed anyway) I could conceivably get interested in moving to sbt-ci-release

**Comment by SethTisue** [comment]
on another PR I wrote the following, which I'll also put here to help clarify what's going on:

> Nobody is in charge of this repo anymore, including me -- see #2183. so we're trying to scrape by with whatever volunteer effort is available. Perhaps this new situation will be at least minimally self-sustaining, perhaps not, I don't know.

and about closing old PRs for inactivity, I wrote:

> right now we have a great many open PRs and not many people volunteering to review them. one problem is that it isn't clear which PRs are even still of interest. I'm trying to cut the PR queue down so that it only contains PRs that someone has recently expressed interest in.

> if you think this PR is mergeable, please rebase it, and if it still passes CI, perhaps a reviewer will appear. (I know nothing about this codebase.)

> or, if you think the PR might still have value but you don't have the time or inclination to rebase it yourself, then let's close it again -- but perhaps these comments will prompt someone else to pick it up.

**Comment by hvesalai** [comment]
If we get the release process to a push-a-button level, then I don't mind my name being mentioned as a person who can push it. But it should not in any form indicate that I would make bug fixes for others. We are using slick in our organization with psql, but it has been really stable for years.

File: README.md [hunk 1]
```diff
@@ -31,4 +31,12 @@ Accessing other database systems is possible, with a reduced feature set.
...
 found on the [Slick web site](http://slick.lightbend.com/).
 
-Licensing conditions (BSD-style) can be found in LICENSE.txt.
+## Maintenance status
+
+Slick is community-maintained.  The lead volunteer maintainer is currently
+@hvesalai.  Lightbend staff (such as @SethTisue) may be able to assist
+with administrative issues.
+
+Slick is no longer covered under Lightbend commercial support.
```
Comment (adriaanm) [inline README.md:40]: Current LB customers are still covered until Dec 31, 2021. Not sure we need the last paragraph here (documentation duplication). Customers were informed last year.

Comment (SethTisue) [reply README.md]: Okay, I dropped the second paragraph.

**Comment by SethTisue** [comment]
@hvesalai I had wanted to acknowledge your efforts, which are greatly appreciated. But yeah, as soon as you identify somebody as being in charge, everybody thinks everything is that person's job 😱

I took your name out and added a "Please help if you can", to emphasize the collective nature of this effort.

**Comment by SethTisue** [comment]
@hvesalai can you write me at seth dot tisue at lightbend dot com? I don't seem to have your email address, and I want to ask you about a maintainer thing

**Pull request merged**